### ISSUE TITLE
Possible to generate release with updated dependency versions?

### ISSUE THREAD

**Issue by jonegard** [MadSpindel/swish-python#12]
Hi! Nice work on this library! Would it be possible to create a new release with updated dependencies for this project and push it to PyPI? The `schematics` dependency specifically is somewhat out of date. Newer releases of that library contain fixes for things which emit deprecation warnings with the current version and it would be nice to stop getting those. 🙂

**Comment by cthart** [comment]
I don't understand this issue. Schematics is at 2.1.1, which is the version that `pip install swish` pulls in. The requirements.txt was updated before the 1.1 release of swish on PyPI, so I don't understand how you're getting an older version.

**Comment by jonegard** [comment]
Hmm, yes, given that `schematics>=2.0.1` is in `requirements.txt` file the most recent version of schematics should be used. I don't think it was be the cause, but I did use Poetry to manage this dependency. It's been some time since I opened this issue but I do clearly remember getting deprecation warnings which prevented a Python upgrade above 3.9. Either way, since I no longer work for the employer where this was an issue and we anyways removed Swish there's no reason to keep this issue open as far as my needs go. I'll go ahead and close it to keep things clean.
